src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const duplicates = Object.values(robot);
+  const repairedRobot = {};
+
+  if (duplicates.filter((item, index) =>
+    duplicates.indexOf(item) !== index).length) {
+    return null;
+  }
+
+  for (const [key, value] of Object.entries(robot)) {
```

Comment:
why not use 'for in' loop here?

Reply:
I tried - it didn't work (due to incorrect writing), but I already figured it out